packages/react-button/src/components/ToggleButton/useChecked.ts

```diff
@@ -25,7 +25,7 @@ export const useChecked = <TDraftState extends CheckedState>(draftState: TDraftS
   // Ensure the right aria value is set to represent the checked state.
   const isCheckboxTypeRole = role === 'menuitemcheckbox' || role === 'checkbox';
 
-  draftState[isCheckboxTypeRole ? 'aria-checked' : 'aria-pressed'] = checkedValue;
+  draftState[isCheckboxTypeRole ? 'aria-checked' : 'aria-pressed'] = !!checkedValue;
```

Comment:
For screen reader user to understand that button is toggle type `aria-pressed` need to be presented on `button`, even with false value. Otherwise user is not aware it is toggle button and hear it from screen reader like normal/common button.